configuration/contrib/diag-plugins/07_mosquitto.sh

```diff
@@ -28,9 +28,7 @@ if [ -n "$OUTPUT_DIR" ] && [ ! -d "$OUTPUT_DIR" ]; then
 fi
 
 mosquitto_journal() {
-    if command -V journalctl >/dev/null 2>&1; then
-        journalctl -u "mosquitto" -n 1000 --no-pager > "$OUTPUT_DIR/mosquitto-journal.log" 2>&1 ||:
-    fi
+    journalctl -u "mosquitto" -n 1000 --no-pager > "$OUTPUT_DIR/mosquitto-journal.log" 2>&1 ||:
```

Comment:
Why limit the lines to 1000? Why not the whole log? Since it is only collected locally and not uploaded to the cloud or anything, the size doesn't really matter, right? If we're really keen to  limit the size, I'd at least use the `-b` option so that everything from the last restart is available.

Reply:
No specific reason behind. I can remove it

Reply:
Removed `-n 1000` in 09d5ee5394c2638dc2cc048a1d70d454261e2133

Reply:
Sorry my statement was incorrect. You're using a journalctl command so the output MUST be limited, otherwise this could provide logs from a very long time back.